baybe/kernels/kernels.py

```diff
@@ -0,0 +1,17 @@
+"""Collection of kernels."""
+
+from attrs import define, field
+from attrs.validators import in_
+
+from baybe.kernels.base import Kernel
+
+
+@define
+class MaternKernel(Kernel):
+    """A Matern kernel using a smoothness parameter."""
+
+    nu: float = field(converter=float, validator=in_([0.5, 1.5, 2.5]), default=2.5)
```

Comment:
should we add a converter that alos allows strings a la `1/2`, `3/2`, `5/2` ? At least I tend to think of these kernels parameters more in these terms

Reply:
Implemented this via a converter, please resolve if the approach is suitable.

Reply:
keep in mind to add this also to hypothesis strategies

Reply:
Please don't add to the strategy itself but rather add a test to the `alternative_creation` folder (that's the purpose of it). I followed the principle that the strategy should merely span the space of possibilities and are minimal in that sense. I think this keeps the code easier to maintain

Reply:
Done. Added two tests for the string based representation of the corresponding fractions and floats and one strategy for creating `Fraction` objects. Please resolve if this is what you had in mind :)